### ISSUE TITLE
Internal: cuDNN launch failure : input shape ([1,3,80,160])

### ISSUE THREAD

**Issue by Blue-Eagle-10** [mvirgo/MLND-Capstone#32]
When I run draw_detected_lanes.py, I always get an error, the error is as follows, how can I solve it? thank you very much!
![image](https://user-images.githubusercontent.com/24905967/80070341-64617d80-8575-11ea-97e7-b770fce5fbdc.png)

**Comment by Blue-Eagle-10** [comment]
Hi @mvirgo, When I run draw_detected_lanes.py, I always get an error, the error is as follows, how can I solve it? thank you very much!
![image](https://user-images.githubusercontent.com/24905967/80070474-94108580-8575-11ea-8ff0-6276395db698.png)

**Comment by mvirgo** [comment]
First, it may depend on an environment difference if you are using a newer version of TensorFlow. I'm actually hoping to finally create an updated environment for the project this week, but admittedly I've planned for that before without a ton of success.

I *think* this has something to do with a mismatch of TensorFlow, cuDNN and CUDA versions, based on some quick research [here](https://github.com/AlexEMG/DeepLabCut/issues/1) and [here](https://stackoverflow.com/questions/52551296/keras-batchnormalization-layer-internalerror-cudnn-launch-failure), or otherwise potentially a memory issue. I was rather inefficient back in the day by loading the full training file all at once instead of just pulling in the needed image files when desired, so that could also be causing the issue.

**Comment by mvirgo** [comment]
No additional comments, so closing out.
